osal/src/os/posix/src/os-impl-no-module.c

```diff
@@ -20,20 +20,15 @@
 
 /**
  * \file     os-impl-no-module.c
- * \ingroup  posix
- * \author   joseph.p.hickey@nasa.gov
+ * \ingroup  FreeRTOS
  *
  */
 
 /****************************************************************************************
                                     INCLUDE FILES
  ***************************************************************************************/
 
-#include <stdlib.h>
-#include <string.h>
-
 #include <osapi.h>
-#include "os-shared-module.h"
```

Comment:
Should this be removed?

Reply:
yeah, again rtems doesnt have it, but they don't implement anything in this function either